hazelcast/src/main/java/com/hazelcast/instance/HazelcastInstanceProxy.java

```diff
@@ -68,7 +68,7 @@
  * </li>
  * </ol>
  */
-public final class HazelcastInstanceProxy implements HazelcastInstance, SerializationServiceSupport {
+public class HazelcastInstanceProxy implements HazelcastInstance, SerializationServiceSupport {
     protected volatile HazelcastInstanceImpl original;
```

Comment:
Are you planning on extending HazelcastInstanceProxy in Jet?
Removing final here may be a bit tricky & hazardous.